lively.components/prompts.cp.js

```diff
@@ -12,11 +12,18 @@ export class AbstractPromptModel extends ViewModel {
       _isActive: { defaultValue: false },
       autoRemove: { defaultValue: true },
       answer: { defaultValue: null, derived: true },
-      label: {
+      title: {
         defaultValue: 'A prompt title',
-        set (label) {
-          this.setProperty('label', label);
-          if (this.view) { this.ui.promptTitle.value = label; }
+        set (title) {
+          this.setProperty('title', title);
+          if (this.view) { this.ui.promptTitle.value = title; }
+        }
+      },
+      text: {
+        defaultValue: null,
+        set (text) {
+          this.setProperty('text', text);
+          if (this.view) this.ui.promptText.value = text;
```

Comment:
The more canonical way to update in response to prop changes is to handle all of this inside the `onRefresh(changedPropName)` method and check for the prop name.

The idea here is to have a central space where all UI updates in response to property changes is handled instead of having them dispersed among a variety of setters inside the property definitions.